convert/svg/fromSvgPath.js

```diff
@@ -51,7 +51,7 @@ const toSegments = ({ curveSegments, tolerance = 0.01 }, svgPath) => {
           ]
         );
         for (const [x, y] of simplifyPath(path, tolerance)) {
-          appendPoint([x, y]);
+          appendPoint([x, y, 0]);
```

Comment:
Are all points 3d now?  And projected to planes when needed? I remember talking about it, but I don't remember what the decision was.

Reply:
It's a bit inconsistent. They should all effectively be 3D.

Points on planes should have an effective Z of 0.